src/function_lua.c

```diff
@@ -480,6 +486,7 @@ int luaEngineInitEngine(void) {
         .get_function_memory_overhead = luaEngineFunctionMemoryOverhead,
         .get_engine_memory_overhead = luaEngineMemoryOverhead,
         .free_function = luaEngineFreeFunction,
+        .gc_step = luaEnglineGCStep,
```

Comment:
Maybe we should call it engine _cron or just cron? Maybe GC is to specific and not all engines will need GC?